bootstrap/inmemory/network.go

```diff
@@ -72,6 +72,7 @@ func NewNetworkWithNumOfNodes(validators map[string]config.ValidatorNode, nodeOr
 		validatorNode := validators[address.KeyForMap()]
 		cfg := cfgTemplate.ForNode(address, privateKeys[address.KeyForMap()])
 		metricRegistry := metric.NewRegistry()
+		metric.RegisterConfigIndicators(metricRegistry, cfg)
```

Comment:
why is this needed? here?

Reply:
Because this is the place where the initialization happens.

I want to report the version right after we start the http server, because otherwise it will be blocked by initialization of other services and it will take some time.

Reply:
I guess adding the values directly to `dumpMetricsAsJSON` should be good enough, however, it would be also a good idea to split metrics and config values that we want to expose

Reply:
I agree about splitting runtime metrics and config values - i wanted to suggest this too but I don't know who else is relying on these "metrics" so if we remove them form the metrics url it may break something... for instance, do we show version number or public addresses in graphana?

Reply:
for instance, the status page shows the version.. that's one place that is relying on the metrics page for versions

Reply:
maybe the simplest way is to leave what existed as it was - and create another config handler in the HTTP server for your needs?

Reply:
The simplest way is indeed to leave what we have now but update `dumpMetricsAsJSON`.

We do not show text values in grafana because Prometheus rightfully does not believe that text is a metric and they simply won't allow it.